doc/dev/zoned-storage.rst

```diff
@@ -0,0 +1,114 @@
+=======================
+ Zoned Storage Support
+=======================
+
+http://zonedstorage.io
+
+Zoned Storage is a class of storage devices that enables host and storage
+devices to cooperate to achieve higher storage capacities, increased throughput,
+and lower latencies. The zoned storage interface is available through the SCSI
+Zoned Block Commands (ZBC) and Zoned Device ATA Command Set (ZAC) standards on
+Shingled Magnetic Recording (SMR) hard disks today and is also being adopted for
+NVMe Solid State Disks with the upcoming NVMe Zoned Namespaces (ZNS) standard.
+
+This project aims to enable Ceph to work on zoned storage drives and at the same
+time explore research problems related to adopting this new interface.  The
+first target is to enable non-ovewrite workloads (e.g. RGW) on host-managed SMR
+(HM-SMR) drives and explore cleaning (garbage collection) policies.  HM-SMR
+drives are high capacity hard drives with the ZBC/ZAC interface.  The longer
+term goal is to support ZNS SSDs, as they become available, as well as overwrite
+workloads.
+
+The first patch in these series enables writing data to HM-SMR drives.  Upcoming
+patches will introduce a rudimentary cleaner to form a baseline for further
+research.  Currently we can perform basic RADOS benchmarks on HM-SMR drives, as
+can be seen below.
+
+Please contact Abutalib Aghayev <agayev@cs.cmu.edu> for questions.
+
+$ zbc_info /dev/sdc
+Device /dev/sdc:
+    Vendor ID: ATA HGST HSH721414AL T240
```

Comment:
```
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/zoned-storage.rst:31: WARNING: Unexpected indentation.
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/zoned-storage.rst:40: WARNING: Block quote ends without a blank line; unexpected unindent.
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/zoned-storage.rst:61: WARNING: Unexpected indentation.
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/zoned-storage.rst:72: WARNING: Block quote ends without a blank line; unexpected unindent.
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/zoned-storage.rst:73: WARNING: Block quote ends without a blank line; unexpected unindent.
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/zoned-storage.rst:91: WARNING: Unexpected indentation.
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/zoned-storage.rst:102: WARNING: Block quote ends without a blank line; unexpected unindent.
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/zoned-storage.rst:103: WARNING: Block quote ends without a blank line; unexpected unindent.
looking for now-outdated files... none found
```

Reply:
My bad!  I forgot to update my local branch and it doesn't contain all the previous changes.  Let me fix that!

Reply:
Done now.  Sorry for the mess up.